packages/cmr-client/docs/API.md

```diff
@@ -0,0 +1,258 @@
+# @cumulus/cmr-client API Documentation
```

Comment:
We should be able to put these docs directly in the `@cumulus/cmr-client/README.md` file.